packages/code-studio/src/styleguide/Inputs.tsx

```diff
@@ -85,6 +85,10 @@ function Inputs(): React.ReactElement {
     []
   );
 
+  const handleChange = (date: string) => {
```

Comment:
We've still got linter issues here - our formatting is fairly strict, we include some extensions and VS Code settings for automatically formatting code on save: https://github.com/deephaven/web-client-ui/blob/main/.vscode/settings.json

For this case though, I'm going to recommend the following:
- Make the `onChange` prop on `DateInput` optional still (the ticket was specifically for the `date` parameter on the callback to be marked as required, we don't need to make `onChange` itself required as well)
- Don't add an `onChange` here in the styleguide.

That would make it consistent with our existing behaviour/other components (such as `DateTimeInput`).